VideoWeb/VideoWeb/ClientApp/src/app/shared/contact-information.ts

```diff
@@ -1,4 +1,10 @@
 export const vhContactDetails = {
-    phone: '0300 303 0655',
-    adminEmail: 'video-hearings@justice.gov.uk'
+    uk: {
```

Comment:
Might be minor but could we change this to englandAndWales?

Reply:
Or maybe they should be hmcts and scts.

Reply:
@FarronF I've updated it to englandAndWales to keep it consistent